include/mata/nfa.hh

```diff
@@ -233,6 +233,9 @@ using TransitionList = Mata::Util::OrdVector<TransSymbolStates>;
 /// Transition relation for an NFA. Each index 'i' to the vector represents a state 'i' in the automaton.
 using TransitionRelation = std::vector<TransitionList>;
 
+/// An epsilon symbol which is now defined as the maximal value of data type used for symbols
+const Symbol EPSILON = limits.maxSymbol;
+
```

Comment:
The `Limits` struct should be updated to use proper limits from `<limits>` header. We now use C limits and long max instead of unsigned long max at that. Furthermore, this constant should be then `constexpr` instead of `const`.

```cpp
static constexpr struct Limits {
    State maxState = std::numeric_limits<State>::max();
    State minState = std::numeric_limits<State>::min();
    Symbol maxSymbol = std::numeric_limits<Symbol>::max();
    Symbol minSymbol = std::numeric_limits<Symbol>::min()
} limits;
```

Reply:
done